tests/cross_functional/ui/fusion_access/test_create_filesystem.py

```
@@ -0,0 +1,233 @@
+        logger.info(f"Step 4a: Enter Image registry URL: {image_registry_url}")
+        self.fusion_access.enter_image_registry_url(image_registry_url)
+        self.base_ui.take_screenshot("image_registry_url_entered")
+
+        # Step 4b: Enter Image repository name
+        image_repository_name = config.ENV_DATA.get("san_image_repository_name")
+        logger.info(f"Step 4b: Enter Image repository name: {image_repository_name}")
+        self.fusion_access.enter_image_repository_name(image_repository_name)
+        self.base_ui.take_screenshot("image_repository_name_entered")
+
+        # Step 4c: Create docker-registry secret in ibm-spectrum-scale namespace
+        secret_name = "quayio-secret"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "secret_name"

(S105)